code/core/src/shared/utils/module.ts

```diff
@@ -112,13 +113,17 @@ export const safeResolveModule = ({
   parent?: string;
   extensions?: string[];
 }) => {
-  for (const extension of [''].concat(extensions)) {
-    try {
-      const resolvedPath = fileURLToPath(importMetaResolve(specifier + extension, parent));
-      if (statSync(resolvedPath).isFile()) {
-        return resolvedPath;
-      }
-      // eslint-disable-next-line @typescript-eslint/no-unused-vars
-    } catch (e) {}
+  try {
+    const resolvedPath = resolveModulePath(specifier, {
+      from: parent,
+      extensions: [''].concat(extensions),
+    });
+    if (statSync(resolvedPath).isFile()) {
+      return resolvedPath;
+    }
+  } catch (e: any) {
+    if (e.code !== 'ERR_MODULE_NOT_FOUND') {
+      throw e;
+    }
```

Comment:
This is the cause of the error @mrginglymus 

Can you explain why you added this?

Reply:
I've removed the code, for now.